students/17th/team1/jiyoonlee/user/views.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from django.http import JsonResponse
+from django.views import View
+
+from .models import User
+
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            mobile_number = data['mobile_number']
+            email         = data['email']
+            full_name     = data['full_name']
+            username      = data['username']
+            password      = data['password']
+
+            if mobile_number is None and email is None:
+                return JsonResponse(
+                    {'message': '휴대폰 번호나 이메일을 입력하세요.'}, 
```

Comment:
```suggestion
                    {'message': 'INVAILD_INPUT'}, 
```
친절한 응답이지만 우리의 메세지는 고객에게 바로 전달하는것이 아닌 프론트 클라이언트에게 전달합니다. 사실은 status code만으로도 응답은 충분합니다. 하지만, 정확한 상황 판단을 위해 약속된 메세지 코드를 주고 받음으로서 상황에 대해 더 명확히 판단할 수 있으므로, 메세지를 주는것 입니다. 그러므로 이러한 문자열은 단순히 메세지가 아니라 문자로 된 상태 코드여야 합니다. 따라서 'INVALID_INPUT' 과 같은 상태코드문자열을 주는게 맞습니다.

Reply:
아래의 다른 리턴에 적힌 메세지도 변경해주세요.

Reply:
수정했습니다